wallet/src/de/schildbach/wallet/util/WalletUtils.java

```diff
@@ -117,11 +118,16 @@ public static Editable formatHash(@Nullable final String prefix, final String ad
     }
 
     public static Wallet restoreWalletFromProtobufOrBase58(final InputStream is,
-            final NetworkParameters expectedNetworkParameters) throws IOException {
+                                                           final NetworkParameters expectedNetworkParameters) throws IOException {
+        return restoreWalletFromProtobufOrBase58(is, expectedNetworkParameters, null);
+    }
```

Comment:
In a future story, we could refactor this class and its static methods into a Wallet Creation/Management Class that gets injected into the right places.